crypto/evp/pmeth_lib.c

```diff
@@ -23,7 +23,7 @@
 #include <openssl/dh.h>
 #include <openssl/rsa.h>
 #include <openssl/kdf.h>
-#include "internal/cryptlib.h"
+#include "crypto/cryptlib.h"
```

Comment:
This change isn't necessary anymore, or is it?

Reply:
`OPENSSL_INIT_BASE_ONLY` is only defined in "crypto/cryptlib.h"

Reply:
Ah, OK.